src/views/MediaChannels/components/MoreInfoDrawer.tsx

```diff
@@ -19,37 +91,212 @@ interface MoreInfoDrawerProps {
   onClose: () => void;
 }
 
+function isValidHttpUrl(string) {
```

Comment:
Would it better to move this to utilities?

Reply:
Yes, you're right. Moved!